backend/app/expenses/service.py

```
@@ -1,13 +1,17 @@
+from fastapi import HTTPException, status
```

Ruff (0.12.2)

`fastapi.status` imported but unused

Remove unused import: `fastapi.status`

(F401)